CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 Changelog
 =========
 
-[next](https://github.com/rigetti/pyquil/compare/master..rc) (In development)
+[next](https://github.com/rigetti/pyquil/compare/v3.0.0..master) (In development)
```

Comment:
This should remain as it was (`v3.0.0..rc` is also valid but with `master..rc` you won't have to keep updating it)